code/ui/.storybook/preview-head.html

```diff
@@ -0,0 +1,4 @@
+<script>
+  // eslint-disable-next-line no-undef
+  window.global = window;
```

Comment:
Let's make it a target for SB7.0 to remove this requirement?

@shilman ☝️ ?

Reply:
FYI this is documented here: https://github.com/storybookjs/storybook/tree/next/code/lib/builder-vite#migration-from-webpack--cra